spec/spec_helper.rb

```diff
@@ -143,6 +143,9 @@ def set_hieradata(hieradata)
     FileUtils.rm_rf(@spec_global_env_temp)
     @spec_global_env_temp = nil
   end
+  c.after(:suite) do
+    RSpec::Puppet::Coverage.report!(100)
```

Comment:
Please don't include code coverage, it breaks under parallel testing (our travis default)